client/app/locales/de.coffee

```diff
@@ -231,12 +231,12 @@ module.exports =
       "message action sent ko"      : "Fehler beim Nachricht senden: "
       "message action draft ok"     : "Nachricht gespeichert"
       "message action draft ko"     : "Fehler beim Nachricht speichern: "
-      "message action delete ok"    : "Nachricht “%{subject}” gelöscht"
-      "message action delete ko"    : "Fehler bei Nachricht löschen: "
-      "message action move ok"      : "Nachricht “%{subject}” wurde verschoben"
-      "message action move ko"      : "Fehler beim verschieben von Nachricht “%{subject}”: "
-      "message action mark ok"      : "Nachricht markiert"
-      "message action mark ko"      : "Fehler beim Nachricht markieren: "
+      "message delete ok"    : "Nachricht “%{subject}” gelöscht"
```

Comment:
MessageActionCreator now handles all action success / failure / undo, so its easier if they have a key of the same form
`type action ko/ok`